crates/apollo-parser/src/parser/grammar/value.rs

```diff
@@ -186,6 +186,48 @@ enum Test @dir__one(string: "string value", int_value: -10, float_value: -1.123e
         }
     }
 
+    #[test]
```

Comment:
Should we add these to the test_data dir? or are these better addressed here?

Reply:
`test_data` only checks the AST/CST resulting from parsing, it doesn't check the conversion from CST StringValue to String.